godot-core/src/log.rs

```diff
@@ -4,6 +4,7 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/.
  */
 
+/// Throws a godot warning
```

Comment:
For this and the others, as they are equivalent to some godot-functions, you could add a 
```
/// _Godot equivalent: @GlobalScope.push_warning()_
```
at the bottom. We do this in some other places too to make it easier for people familiar with godot.

Reply:
Done